Resources/Prototypes/_Harmony/StatusIcon/security.yml

```
@@ -11,4 +11,4 @@
   id: SecurityIconSearch
   icon:
     sprite: /Textures/_Harmony/Interface/Misc/security_icons.rsi
-    state: hud_search
+    state: hud_search
```

Missing new line at the end of file.
```suggestion
    state: hud_search

```